draft-thomson-quic-tls.md

```diff
@@ -416,6 +436,8 @@ Server Write IV with the sequence numbers.  The 48 bits of the reconstructed
 QUIC sequence number (see {{seq-num}}) in network byte order is left-padded with
 zeros to the N_MAX parameter of the AEAD (see Section 4 of [RFC5116]).  The
 exclusive OR of the padded sequence number and the IV forms the AEAD nonce.
+TODO: Not sure if this belongs here or not, but in QUIC multipath we also add
+the path id into the nonce.
```

Comment:
Yes, it will need to be here if we use the same keying material on both paths.  I'll leave a marker here as a reminder until multipath is better understood.  

We need to bit pack the path identifier into the upper bits.  That increases the minimum value for N_MAX, but we're only using 48 bits so far, so we could handle a path identifier of up to 48 bits.

I was going to suggesting using the additional authenticated data for the path identifier, but that would be a huge mistake.